modules/core/src/main/scala/Session.scala

```diff
@@ -252,10 +253,11 @@ object Session {
     writeTimeout: FiniteDuration = 5.seconds,
     strategy:     Typer.Strategy = Typer.Strategy.BuiltinsOnly,
     ssl:          SSL            = SSL.None,
+    connProps: Map[String, String] = StartupMessage.DefaultConnectionParameters
   ): Resource[F, Resource[F, Session[F]]] = {
 
     def session(socketGroup:  SocketGroup, sslOp: Option[SSLNegotiation.Options[F]]): Resource[F, Session[F]] =
-      fromSocketGroup[F](socketGroup, host, port, user, database, password, debug, readTimeout, writeTimeout, strategy, sslOp)
+      fromSocketGroup[F](socketGroup, host, port, user, database, password, debug, readTimeout, writeTimeout, strategy, sslOp, connProps)
```

Comment:
Let's rename `connProps` to `parameters` everywhere. Postgres is very consistent about this terminology so we should use it as well.